src/lib/components/Navbar.svelte

```diff
@@ -89,86 +88,98 @@ The navbar component.
             </svelte:fragment>
         </Dropdown>
         <!-- Book Selector -->
-        <Dropdown>
-            <svelte:fragment slot="label">
-                {nextRef.book}
-                <DropdownIcon />
-            </svelte:fragment>
-            <svelte:fragment slot="content">
-                <TabsMenu
-                    bind:this={bookSelector}
-                    options={{
-                        Book: {
-                            component: SelectGrid,
-                            /**
-                             * TODO:
-                             * - add book abbreviations to catalog to be used in UI instead of bookCode
-                             */
-                            props: { options: books.map((b) => b.bookCode) /*bookAbbreviations*/ }
-                        },
-                        Chapter: {
-                            component: SelectGrid,
-                            props: { options: Object.keys(chapters) }
-                        },
-                        Verse: {
-                            component: SelectGrid,
-                            props: { options: Object.keys(chapters[nextRef.chapter]) }
-                        }
-                    }}
-                    active="Book"
-                    on:menuaction={navigateReference}
-                />
-            </svelte:fragment>
-        </Dropdown>
+        {#if $globalConfig.mainFeatures['book-select'] === 'grid' || $globalConfig.mainFeatures['book-select'] === 'list'}
+            <Dropdown>
```

Comment:
Please change this so that it is:

```
{#if  $globalConfig.mainFeatures['book-select'] === 'grid' }
<Dropdown>
...
</Dropdown>
{:else if $globalConfig.mainFeatures['book-select'] === 'list' }
<! -- TODO: Add List selector -->
{/if}

Reply:
done